HW1/P6/P6.py

```diff
@@ -0,0 +1,56 @@
+import findspark
+findspark.init()
+import pyspark
+from scipy import stats
+import numpy as np
+import matplotlib.pyplot as plt
+sc = pyspark.SparkContext(appName="P6")
+
+if __name__ == "__main__":
+    textRDD = sc.textFile('Shakespeare.txt')
+    wordRDD = textRDD.flatMap(lambda x: x.split(' '))
+    filteredRDD = wordRDD.filter(lambda x: x.isdigit() == False).\
```

Comment:
This takes off all the uppercase letters. Including the ones with a period. The only upper case letters are the characters which he wanted us to remove. 
Yeah I ll keep that in mind when applying filters. Thanks!